### PR TITLE
add file chn_translation_arima_models.Rmd

### PR DESCRIPTION
**Pull request by kguo98**

### CODE REVIEW

**Comment by anyunshan** [comment]
Thanks for the contribution!

File: chn_translation_arima_models.Rmd [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+虽然指数平滑方法不对时间序列中的相邻数据作假设，在某些情况下你可以通过考虑数据间的相关性来做出更好的预测模型。差分整合移动平均自回归模型（ARIMA）包含了一个允许异常成分中非零自相关性的统计模型。
+
+### 差异化一组时间序列
+
+ARIMA模型是为平稳时间序列定义的。所以如果需要处理一组非平稳时间序列，你首先需要‘差异化’时间序列直至获得一组平稳时间序列。如果你需要差异化d次来获得平稳时间序列，那你就会用ARIMA(p,d,q)模型，d代表差异次数。
+
+你可以在R中使用‘diff()’函数来差异化时间序列。比如，1866年至1911年女士裙子的直径均值上下浮动且不平稳：
+
+![](pics/1.png)
```
Comment (kennylee15) [inline chn_translation_arima_models.Rmd:19]: The location of the folder and file is incorrect. Other markdown links have the same issue.

**Review by kennylee15 — COMMENTED** [review]
The location of the folder where image files are stored is incorrect.

**Comment by kennylee15** [comment]
There is a problem with markdown links. The images are not loading properly in the deployed version (https://jtr13.github.io/cc21/chinese-translation-of-using-arima-models-in-r.html). Could you correctly specify the location of the file? They should be under resources.

```
resources/your-project-name/filename
```

Since it has already been merged, please create a new pull request with corrected links. See the link below to see step-by-step instructions on how to do that.

https://edav.info/contribute.html#github-only-walkthrough

Thanks!

**Pull request merged**